kinto_client/__init__.py

```diff
@@ -140,7 +140,16 @@ def clone(self, **kwargs):
     def batch(self, **kwargs):
         batch = Batch(self)
         yield self.clone(session=batch, **kwargs)
-        batch.send()
+        resp, headers = batch.send()
+        for i, response in enumerate(resp['responses']):
+            status_code = response['status']
+            if not (200 <= status_code < 400):
```

Comment:
We probably should factorize this code somehow, it's repeated from elsewhere right?